picoquic/tls_api.c

```diff
@@ -1058,7 +1058,7 @@ void * picoquic_pn_enc_create(
 {
     int ret = 0;
     uint8_t key[256]; 
-    ptls_cipher_context_t *pn_enc = ptls_cipher_new(aead->ctr_cipher, 1, key);
+    ptls_cipher_context_t *pn_enc;
 
```

Comment:
You need to set pn_sec = NULL. Otherwise, if picoquic_hkdf_expand_label fails, the function returns an unitialized pointer.